addons/web/static/src/views/relational_model.js

```diff
@@ -1325,7 +1325,7 @@ export class Record extends DataPoint {
             activeField &&
             !this._isInvisible(fieldName) &&
             value &&
-            (!value[1] || (activeField.options || {}).always_reload)
+            (value[1] === undefined || (activeField.options || {}).always_reload)
```

Comment:
this one should be tested (and the test can be forwardported because nothing breaks in master if we remove the undefined check)